Makefile

```diff
@@ -51,37 +55,43 @@ vendor:
 dep:
 	go mod download
 
-.PHONY: binaries
-binaries:
-	gox -os="linux darwin windows" -arch="amd64" -output="dist/smi_metrics_{{.OS}}_{{.Arch}}" ./cmd/smi-metrics
+tmp:
+	mkdir tmp
+
+.PHONY: build-chart
+build-chart: tmp release-bootstrap
+ifndef CIRCLE_TAG
+	@echo "Missing CIRCLE_TAG, is this being run from circleci?"
+	@exit 1
+endif
+	cp -R chart tmp/smi-metrics
+	sed -i.bak 's/CHART_VERSION/${VERSION}/g' tmp/smi-metrics/Chart.yaml
+	helm package tmp/smi-metrics -d tmp --save=false
+	rm -rf tmp/smi-metrics/
 
 .PHONY: dev
 dev: bootstrap
 	tilt up
 
 .PHONY: release
-release: release-bootstrap
-	set +e \
-    $(eval RELEASE_TAG=$(shell semantics --output-tag))
-	@if [ $(RELEASE_TAG) ]; then \
-	  echo "Tagging the latest commit image with tag: ${RELEASE_TAG}"; \
-	  docker tag ${IMAGE}:${COMMIT_TAG} ${IMAGE}:${RELEASE_TAG}; \
-	  echo "Pushing the docker image: ${IMAGE}:${RELEASE_TAG} "; \
-	  docker push ${IMAGE}:${RELEASE_TAG}; \
-	  echo "Generating binaries for Github Release"; \
-	  env GO111MODULE=on make vendor; \
-	  make binaries; \
```

Comment:
how come you deleted the `make binaries` step?

Reply:
Felt a little weird to upload binaries only used inside the docker image that we already pushed.

Reply:
ah yea 👍